packages/avatar-group/test/dom/__snapshots__/avatar-group.test.snap.js

```diff
@@ -148,8 +146,8 @@ snapshots["vaadin-avatar-group theme"] =
   id="overlay"
   no-vertical-overlap=""
 >
-  <template>
-  </template>
+  <vaadin-list-box role="list">
+  </vaadin-list-box>
```

Comment:
How about we add one more test specifically for the overflowing case?

Reply:
Good point, we can add it later in a separate PR.